src/ui/screens/home/components/project-items-freelancer/project-liked-freelancer.component.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { PureComponent } from "react";
+import { Grid } from "@material-ui/core";
+import GridList from "@material-ui/core/GridList";
+import GridListTile from "@material-ui/core/GridListTile";
+import ListSubheader from "@material-ui/core/ListSubheader";
+import "./project-items-freelancer.css";
+import ProjectService from "../../../../../service/project.service";
+import { AxiosError, AxiosResponse } from "axios";
+import IconButton from "@material-ui/core/IconButton";
+import { ContentWrapper } from "components/index";
+
+interface IProjectLikedFreelancerPropType {}
+
+interface IProjectLikedFreelancerStateType {
+  data: any;
+  error: boolean;
+}
+
+export class ProjectLikedFreelancer extends PureComponent<
+  IProjectLikedFreelancerPropType,
+  IProjectLikedFreelancerStateType
+> {
+  constructor(props: IProjectLikedFreelancerPropType) {
+    super(props);
+
+    this.state = {
+      data: [],
+      error: false
+    };
+
+    this.renderItemProject = this.renderItemProject.bind(this);
+    this.removeInterestOnProject = this.removeInterestOnProject.bind(this);
+    this.renderInterestsProjectsData = this.renderInterestsProjectsData.bind(this);
```

Comment:
Set here the variable for current path into local storage to maintain the underlined selected menu.